components/04-molecules/article&blog-post-elements/author-bio/_author-bio.scss

```diff
@@ -1,6 +1,7 @@
 $author-bio-image-offset: ms(22) / 2;
 // tiny space between name and info
 $author-text-margin: 0.18rem;
+$author-name-xl: $font-size-base*1.55 ; //~20px
```

Comment:
Bitte keine Multiplikation hier verwenden. Es gibt einige vorgefertigte font-sizes, die du verwenden kannst. Diese findest du in der typoghraphy.scss